specification/marketplacecatalog/data-plane/Search/stable/2022-02-02/search.json

```diff
@@ -0,0 +1,1065 @@
+{
+  "x-generator": "NSwag v13.15.10.0 (NJsonSchema v10.6.10.0 (Newtonsoft.Json v13.0.0.0))",
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft Marketplace Search API",
+    "version": "2022-02-02"
+  },
+  "host": "catalogapi.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "produces": [
+    "text/plain",
+    "application/json",
+    "text/json"
+  ],
+  "paths": {
+    "/autocomplete": {
+      "get": {
+        "tags": [
+          "Public-Search"
+        ],
+        "summary": "Get a list of complete terms based on partial search term",
+        "operationId": "AutoComplete_Get",
+        "x-ms-examples": {
+          "AutoComplete_Get": {
+            "$ref": "./examples/AutoComplete_Get.json"
+          }
+        },
+        "parameters": [
+          {
+            "type": "string",
+            "name": "searchQuery",
+            "in": "query",
+            "x-nullable": true
```

Comment:
Don't use of `x-nullable: true`. Properties or parameters with no value should simply be omitted from the payload. You should eliminate all instances of `x-nullable: true` (76 of them).

Reply:
done